Library/DiscUtils.Dmg/UdifBuffer.cs

```diff
@@ -279,14 +279,14 @@ private void LoadRun(CompressedRun run)
                     _stream.Position = run.CompOffset + 2; // 2 byte zlib header
                     using (DeflateStream ds = new DeflateStream(_stream, CompressionMode.Decompress, true))
                     {
-                        StreamUtilities.ReadFully(ds, _decompBuffer, 0, toCopy);
+                        StreamUtilities.ReadExact(ds, _decompBuffer, 0, toCopy);
```

Comment:
Reads a number of bytes per run specification. Did not throw before, does now, and should.